server/start.go

```diff
@@ -245,6 +246,11 @@ func startInProcess(ctx *Context, clientCtx client.Context, appCreator types.App
 		return err
 	}
 
+	config := config.GetConfig(ctx.Viper)
+	if err := config.ValidateBasic(); err != nil {
```

Comment:
We should add a changelog entry, I think this constitues as a breaking change since validators need to manually tweak their app.toml.

Reply:
Yes